.github/workflows/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+name: Molecule Test
+on: [push, pull_request]
+jobs:
+  molecule:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
I've recently started seeing complaints that `v2` is deprecated. From what I saw `v3` is a direct replacement (not confirmed). Might be worth trying v3 out and seeing if it works as-is. We'll need to start going through and bumping to v3 at some point.